src/dune_rules/jsoo/js_of_ocaml.ml

```diff
@@ -91,6 +93,7 @@ module In_buildable = struct
   type t =
     { flags : Ordered_set_lang.Unexpanded.t Flags.t
     ; javascript_files : string list
+    ; wasm_files : string list
```

Comment:
I think it would be nice to have a comment explaining why wasm_of_ocaml reuses the `(js_of_ocaml ..)` field instead of being its own thing